packages/auth/src/grant/routes.ts

```diff
@@ -265,15 +269,19 @@ async function acceptGrant(
     ctx.body = {
       error: 'request_denied'
     }
+    return
```

Comment:
Not related to the changes, but is it ok to use status = 400 above on L268 even though it's not listed as a possible returned status code in the Open API spec?

https://github.com/interledger/open-payments/blob/f365dbec4b9dec98b9f622bc49a92aea9ee01568/openapi/AS/IdP/openapi.yaml#L122-L128

Reply:
Some of this stuff (to my recollection) was written before the OpenAPI spec was brought in, so it's more like an initial stab. There could be something to adding a response to the spec for unexpected stuff, but later I will go over all these responses and decide if they'll conform to an existing response or a new one should be added to the spec.